dask/array/creation.py

```diff
@@ -411,16 +411,17 @@ def indices(dimensions, dtype=int, chunks='auto'):
     return grid
 
 
-def eye(N, chunks, M=None, k=0, dtype=float):
+def eye(N, chunks='auto', M=None, k=0, dtype=float):
     """
     Return a 2-D Array with ones on the diagonal and zeros elsewhere.
 
     Parameters
     ----------
     N : int
       Number of rows in the output.
-    chunks: int
-        chunk size of resulting blocks
+    chunks: int, str
```

Comment:
Copying from `from_array`, we may want

```
    chunks : int, str
        How to chunk the array. Must be one of the following forms:
        -   A blocksize like 1000.
        -   A size in bytes, like "100 MiB" which will choose a uniform
            block-like shape
        -   The word "auto" which acts like the above, but uses a configuration
            value ``array.chunk-size`` for the chunk size
```

I removed the tuple options, as I don't think those will work yet.

Reply:
Thank you for the suggestion, @TomAugspurger! I will update the docstring to this new version.

Reply:
Good suggestion @TomAugspurger. Thanks for handling this @andersy005